src/CRUD.js

```diff
@@ -38,14 +38,40 @@ const changeItem = (value, id) => {
   Storage.SetLocalStorage(arr)
   console.log(arr)
 }
+const markCompleted = (checkbox, id, toDoList) => {
+  const arr = Storage.getLocalStorage()
+  arr[id - 1].completed = checkbox.checked
+  Storage.SetLocalStorage(arr)
+  display(toDoList)
+}
```

Comment:
Please make sure that when a task's checkbox is checked the task's description has a `strike-through` styling.